packages/management-api-app/src/management.tsx

```diff
@@ -32,7 +32,7 @@ export const Management: React.FunctionComponent = () => {
   const [error, setError] = useState<Error | null>()
   const [username, setUsername] = useState('')
 
-  const [pods, setPods] = useState<ManagedPod[]>([])
+  const [pods, setPods] = useState<ManagedPod[] | null>(null)
```

Comment:
Like I said, instead of introducing adhoc `null` type, we should use `isLoading` flag to decide whether it's a valid state of `[]` or just still loading.

Reply:
But I can't see how it's related to Node upgrade.

Reply:
Sneaked into this commit as a result of the fix for the monkey patching. Now removed.